docker-rollout

```diff
@@ -128,6 +131,9 @@ main() {
       docker $DOCKER_ARGS rm $NEW_CONTAINER_IDS
 
       exit 1
+    elif [ "$WAIT_AFTER_HEALTHY_DELAY" != "0" ]; then
```

Comment:
I suggest adding this as a separate conditional, to highlight that previous check halts the program execution.

```suggestion
    fi
    
    if [ "$WAIT_AFTER_HEALTHY_DELAY" != "0" ]; then
```